docs/site/development/development-process/design-proposals/xmb.md

```diff
@@ -108,11 +108,19 @@ Example:
 
 Plurals are represented with ICU Syntax, such as:
 
-\<msg id='4AC13E2DA211C113' desc='[ICU Syntax] The pattern used to compose plural for week, including abbreviated forms. These forms are special! Before translating, see cldr.org/translation/plurals.'
+<!--
+  {% raw %}
 
-\>{LENGTH, select,
+  Disable liquid parsing on this codeblock to prevent errors reading '{{'
+  See: https://talk.jekyllrb.com/t/code-block-is-improperly-handled-and-generates-liquid-syntax-error/7599/2
+-->
 
-abbreviated {{NUMBER\_OF\_WEEKS, plural,
+```xml
```

Comment:
@chpy04 this wasn't needed but i decided to put this in an XML code fence.

The real fix is the `{% raw…` directive.